manifests/stable/supported-nic-ids_v1_configmap.yaml

```diff
@@ -27,6 +27,7 @@ data:
   Qlogic_qede_QL41000: 1077 8070 8090
   Qlogic_qede_QL45000_25G: 1077 1656 1664
   Qlogic_qede_QL45000_50G: 1077 1654 1664
+  Pensando_DSC_25GB_100GB: "1dd8 1002 1003"
```

Comment:
Could you remove the quotes?

Reply:
It seems support for pensando card has already been added to this file. 
See: [https://github.com/openshift/sriov-network-operator/blob/d9a8cbb989c0dcd7decf28a25bc26834336fa5eb/manifests/stable/supported-nic-ids_v1_configmap.yaml#L26](url)

I will just remove this line altogether